_components/customisation.md

```diff
@@ -0,0 +1,199 @@
+---
+title: Component Customisation
+description: Decide if you should customise a component for your project, learn the ways a component may be customised, and avoid other approaches which may be unreliable.
+cta: Read more about how to customise components
+
+# Redirect from previous branding page
+redirect_from:
+  - /docs/components/branding/
+
+# Navigation config
+nav_display: true
+nav_label: Customisation
+nav_order: 11
+---
+
+# {{ page.title }}
+
+Origami aims to improve design consistency and reduce the time teams spend repeating work. Providing component APIs for customisation helps achieve those aims by allowing components to be used across different contexts and different brands, and to provide flexibility for novel projects which may introduce a more unique style.
+
+This document aims to help you decide if you should customise a component for your project, show the ways a component may be customised, and demonstrate why other approaches should be avoided.
+
+<abbr title="Too long; didn't read">
+<strong>
+TL;DR:
+</strong>
+</abbr>
+- Double check customisation is appropriate. Reusing an existing pattern or updating a component to support and share your usecase is preferable.
+- Specify a brand which suits your type of project.
+- Customise component's using their Sass API.
+- Avoid CSS overrides, they are unsupported and unreliable.
+
+## Check Customisation Is Appropriate
+
+The first step before customising an Origami component is to ask:
+1. Is there an existing style which could work already? After searching the [Origami Registry](https://registry.origami.ft.com/components) it's worth double checking in the <a href="https://financialtimes.slack.com/messages/{{site.data.contact.slack}}">#{{site.data.contact.slack}}</a> Slack channel, to reduce duplicated effort and to help maintain a consistent user interface across FT Group projects.
+2. Would your customisation be useful for other teams/projects? Maybe a component should be updated or added to for your usecase, so other teams benefit from your work. A new component might be more appropriate if your design diverges significantly.
+
+A good place to start is to ask for feedback in the <a href="https://financialtimes.slack.com/messages/{{site.data.contact.slack}}">#{{site.data.contact.slack}}</a> Slack channel. The core Origami team, along with the design team, can help identify similar existing styles or help make any relevant component updates to support your usecase without customisation.
+
+## Brands
+
+Most Origami components are branded to provide a distinct appearance within different contexts. A brand may be thought of as a kind of theme, but applies to all components within a project.
```

Comment:
We should likely remove the part which says a brand can be thought of as a kind of theme, because we use the concept of a theme later on in the tutorial when we talk about "Themes/Variants". I would worry using "theme" here would confuse readers